pyrasterframes/src/main/python/docs/raster-processing.md

```diff
@@ -2,8 +2,9 @@
 
 @@@ index
 
-* @ref:[Local Algebra](local-algebra.md)
+* @ref:[Local Map Algebra](local-algebra.md)
 * @ref:["NoData" Handling](nodata-handling.md)
```

Comment:
Should we move this so the "algebras" are near each other?

Reply:
my thinking was that the zonal leans on the same ideas as masking.

But we could make it stand alone more and reordre.